syntax/index.html

```diff
@@ -649,14 +651,20 @@
             Split the header and data rows into cells using the <a>delimiter</a>. Values that are enclosed within the <a>quote character</a> may contain the <a>delimiter</a>. The <a>quote character</a> may be escaped using the <a>escape character</a> where it appears within cells. If the <a>escape character</a> is not the same as the <a>quote character</a> then the <a>escape character</a> is also used to escape the character that immediately follows it.
           </p>
           <p>
-            If <a>trim</a> is <code>true</code> or <code>start</code> then whitespace from the start of values that are not enclosed must be removed from the value. If <a>trim</a> is <code>true</code> or <code>end</code> then whitespace from the end of values that are not enclosed must be removed from the value.
+            If <a>trim</a> is <code>true</code> or <code>start</code> then whitespace from the start of values that are not enclosed MUST be removed from the value. If <a>trim</a> is <code>true</code> or <code>end</code> then whitespace from the end of values that are not enclosed MUST be removed from the value.
           </p>
         </li>
+        <li>
+          Construct <a>embedded metadata</a> as Table Metadata (see <cite><a href="http://www.w3.org/TR/tabular-metadata/#tables"></a></cite> in [[!tabular-metadata]]) using the <code>@context</code> from <a>merged metadata</a> and setting the <code>language</code> annotation to the default language from the <code>@context</code>.</li>
```

Comment:
The default language from the `@context` is for interpreting the natural language strings within the metadata document. It doesn't say anything about the language of the CSV file itself. That's from the `language` property on the table description for the CSV file.